chromadb/test/property/strategies.py

```diff
@@ -359,7 +387,21 @@ def metadata(draw: st.DrawFn, collection: Collection) -> types.Metadata:
         sampling_dict: Dict[str, st.SearchStrategy[Union[str, int, float]]] = {
             k: st.just(v) for k, v in collection.known_metadata_keys.items()
         }
+
         metadata.update(draw(st.fixed_dictionaries({}, optional=sampling_dict)))  # type: ignore
+        blacklist_categories = ("Cc", "Cs")
```

Comment:
In the rest of the tests, we have control and special characters. How does this break the like/nlike tests?

Reply:
The control and special characters interfere with sqlite's processing, as stated within the comments for the `document` function that generates the contains/not_contains tests, which also blacklists the Cc and Cs character categories.